gammapy/image/tests/test_utils.py

```diff
@@ -219,3 +224,46 @@ def test_lon_lat_rectangle_mask():
                                   lon_max=None, lat_min=None,
                                   lat_max=None)
     assert_allclose(mask.sum(), 80601)
+
+
+@requires_data('gammapy-extra')
+def test_bin_events_in_cube():
+    dirname = '$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2'
+    data_store = DataStore.from_dir(dirname)
+
+    events = data_store.load(obs_id=23523, filetype='events')
+
+    # Define WCS reference header
```

Comment:
Could you please move that code into a utility function `gammapy.image.make_cube_header`?

You're already duplicating it here:
https://github.com/gammapy/gammapy-extra/blob/master/experiments/sherpa_cube_analysis_prototype/prepare_data_cubes.py#L26

It could work similarly to the existing `gammapy.image.make_header` (to be renamed to `make_image_header`)
http://gammapy.readthedocs.org/en/latest/api/gammapy.image.make_header.html
and `gammapy.image.make_empty_image`, no?
http://gammapy.readthedocs.org/en/latest/api/gammapy.image.make_empty_image.html